operate/importer/src/main/java/io/camunda/operate/zeebeimport/ImportPositionHolder.java

```diff
@@ -23,12 +23,10 @@
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Qualifier;
-import org.springframework.context.annotation.DependsOn;
 import org.springframework.scheduling.concurrent.ThreadPoolTaskScheduler;
 import org.springframework.stereotype.Component;
 
 @Component
-@DependsOn("schemaStartup")
 public class ImportPositionHolder {
```

Comment:
❓ this will now `scheduleImportPositionUpdateTask` without waiting for `schemaStartup`. Do we need that?

Reply:
Right now on the tests I see failing tests logs of missing `operate-import-position`, and it those logs trace back to `ImportPositionHolder.java:112`  which gets called from `ElasticsearchImportStore.java:77`.

This probably won't fail for an already running importer, but it will fail if we run it on an upgraded cluster. since we won't have this index anymore, right? Or will we still keep tracking of (then `exporting`) position?

I guess the class itself need to stay for upgrading clusters that still need to import records from older versions (before 8.7). But I'm not 100% sure.